### PR TITLE
Add an ola_trigger config to control relays using crelay

### PR DESCRIPTION
**Pull request by gobo-ws**
Add an ola_trigger config to control relays using crelay - https://github.com/ondrej1024/crelay/

### CODE REVIEW

**Comment by gobo-ws** [comment]
Hi @peternewman ,
Is it ok now? I am still learning Github.

File: AUTHORS [hunk 1]
```diff
@@ -14,7 +14,7 @@ Contributors:
   Heikki Junnila, bug fixes for the debian packaging files
   Laurent (Renzo), Debian packages, FreeBSD & RDM testing.
   Lukas Erlinghagen, win32 port.
-  Johan Nilsson, Philips Hue trigger config
+  Johan Nilsson, Philips Hue & crelay trigger config
```
Comment (peternewman) [inline AUTHORS]: Up to you, but you may want to change the Philips Hue bit to include Osram Lightify too now?

File: tools/ola_trigger/Makefile.mk [hunk 2]
```diff
@@ -6,6 +6,7 @@ dist_noinst_DATA += \
     tools/ola_trigger/contrib/mac_itunes.conf \
```
Comment (peternewman) [inline tools/ola_trigger/Makefile.mk]: Could we have these in order, so crelay at the top please?

File: tools/ola_trigger/Makefile.mk [hunk 3]
```diff
@@ -73,7 +74,7 @@ tools_ola_trigger_ActionTester_LDADD = $(COMMON_TESTING_LIBS) \
...
-	echo "for FILE in ${srcdir}/tools/ola_trigger/example.conf ${srcdir}/tools/ola_trigger/test_file.conf ${srcdir}/tools/ola_trigger/contrib/mac_volume.conf ${srcdir}/tools/ola_trigger/contrib/mac_itunes.conf ${srcdir}/tools/ola_trigger/contrib/philips_hue_osram_lightify.conf; do echo \"Checking \$$FILE\"; ${top_builddir}/tools/ola_trigger/ola_trigger${EXEEXT} --validate \$$FILE; STATUS=\$$?; if [ \$$STATUS -ne 0 ]; then echo \"FAIL: \$$FILE caused ola_trigger to exit with status \$$STATUS\"; exit \$$STATUS; fi; done; exit 0" > tools/ola_trigger/FileValidateTest.sh
+	echo "for FILE in ${srcdir}/tools/ola_trigger/example.conf ${srcdir}/tools/ola_trigger/test_file.conf ${srcdir}/tools/ola_trigger/contrib/mac_volume.conf ${srcdir}/tools/ola_trigger/contrib/mac_itunes.conf ${srcdir}/tools/ola_trigger/contrib/philips_hue_osram_lightify.conf ${srcdir}/tools/ola_trigger/contrib/crelay.conf; do echo \"Checking \$$FILE\"; ${top_builddir}/tools/ola_trigger/ola_trigger${EXEEXT} --validate \$$FILE; STATUS=\$$?; if [ \$$STATUS -ne 0 ]; then echo \"FAIL: \$$FILE caused ola_trigger to exit with status \$$STATUS\"; exit \$$STATUS; fi; done; exit 0" > tools/ola_trigger/FileValidateTest.sh
```
Comment (peternewman) [inline tools/ola_trigger/Makefile.mk]: Could you update this to match above please

**Comment by peternewman** [comment]
That's as intended now @gobo-ws . No worries about it, GitHub seems to make it very easy to do the first edit, but harder to do subsequent ones, I've certainly been caught out in the same way in the past.

File: AUTHORS (hunk 1 above)
Comment (gobo-ws) [inline AUTHORS]: Yeah good point. Updated now.

File: tools/ola_trigger/Makefile.mk (hunk 2 above)
Comment (gobo-ws) [inline tools/ola_trigger/Makefile.mk]: Fixed now

**Comment by peternewman** [comment]
Retrospective review if you fancy it @nomis52 .

**Pull request merged**